tests/unit/scan.tcl

```diff
@@ -109,7 +109,7 @@ proc test_scan {type} {
 
         after 2
 
-        # TODO: remove this in redis 8.0
+        # TODO: remove this in server version 8.0
```

Comment:
Are these TODOs covered by that issue you created for some other TODOs? I didn't check. There are a few of these in this test suite.

Reply:
Yeah I believe these are tests related to the proposed SCAN changes